### ISSUE TITLE
Eliminate Xcode warnings on generated code

### ISSUE THREAD

**Issue by atticus183** [icerockdev/moko-kswift#46]
It seems 0.5.0 introduced the following Xcode warning:

```
Forced cast from 'Class.dataclass1' to 'Class.dataclass2' always succeeds
```

I propose removing the bang operator from the `SealedToSwiftEnumFeature.kt` file, specifically inside the `buildSealedProperty` method.

I will happily fork and create the PR myself 👍🏼.

**Comment by Alex009** [comment]
will be released in 0.6.0

**Comment by sebj** [comment]
I've just started looking at this library, but I think I'm running into this issue even though it's supposed to be fixed in 0.6.0. I'm seeing 125 Xcode warnings inside my generated Swift file for `forced cast from '..' to '..' always succeeds, did you mean to use 'as'?`. 

My environment:
* KSwift version 0.6.0
* Xcode 13.4.1 (13F100)
* Targeting iOS 14.1+
* Manually integrating the generated Swift files rather than using Cocoapods

**Comment by hoc081098** [comment]
Version 0.6.0 still have this issue
